component/publisher-client/src/test/java/org/wso2/am/analytics/publisher/DefaultFaultMetricBuilderTestCase.java

```diff
@@ -86,6 +86,7 @@ public void testMetricBuilder() throws MetricCreationException, MetricReportingE
                 .addAttribute(Constants.ERROR_CODE, 401)
                 .addAttribute(Constants.ERROR_MESSAGE, "Authentication Error")
                 .addAttribute(Constants.API_ID, "9876-54f1")
+                .addAttribute(Constants.API_TYPE, "HTTP")
```

Comment:
Shall we change this to REST. I hope real values don't have an apiType as "HTTP"

Reply:
anyway, this is only for test. But APIM set type like this. By default(rest API) are named as HTTP. but for others graphql and WS are there.

Reply:
ok, understood.